packages/manager/modules/pci/src/projects/new/payment/components/choose/index.scss

```diff
@@ -0,0 +1,76 @@
+.pci-project-new-payment-choose {
```

Comment:
Need to override some styles from ui-kit/ods in order to fit as more as possible the mockups

Reply:
I can understand that some overriding is needed now and then, but shouldn't the latest mockup design then be integrated in ui-kit/ods and therefore the UI changes automatically applied ?

Reply:
That should be the rule indeed and I prefer this approach too